authorize-ios.js

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+'use strict';
+
+var authorize = require('./build/index.js');
+
+authorize();
```

Comment:
I feel that this should be `require('asyncbox').asyncify(authorize)`, otherwise error messages will be lost instead of printed to the console